app.js

```diff
@@ -1,5 +1,151 @@
 'use strict';
+var storeHours = ['6am', '7am', '8am', '9am' , '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+//var counter = 0;
+var pikeStore = document.getElementById('andPike');
+var seaTacStore = document.getElementById('seaTac');
+var seaCentStore = document.getElementById('seattleCenter');
+var capHillStore = document.getElementById('capitolHill');
+var alkiStore = document.getElementById('alki');
 
-var pikeLocation = {
-  
-}
+var pikeLoc = {
+  minCust: 23,
+  maxCust: 65,
+  avCookie: 6.3,
+  results: [],
+  rndCust: function() {
+    return Math.floor((Math.random() * (this.maxCust - this.minCust + 1))+this.minCust);
+
+  },
+
+  storeCalc: function(){
+    var counter = 0;
+    for (var i = 0; i < 15; i++){
+      var averageCalc = Math.round(this.avCookie * this.rndCust());
+      this.results.push(averageCalc);
+      var liEl = document.createElement('li');
+      liEl.textContent = storeHours[i] + ': ' + pikeLoc.results[i] + ' cookies.';
+      pikeStore.appendChild(liEl);
+      //var counter = pikeLoc.results[i] + pikeLoc.results[0];
+      counter += pikeLoc.results[i]
+      var totCook = document.createElement('li');
+      totCook.textContent = 'Total cookies: ' + counter;
+
```

Comment:
you only ever need one space between lines of code (and that's normally only to separate groups so things are more readable!)